.circleci/config.yml

```diff
@@ -11,13 +11,13 @@ references:
     working_directory: ~/addons-frontend
     docker:
       # This is the NodeJS version we run in production.
-      - image: circleci/node:12
+      - image: circleci/node:14
```

Comment:
were we already running node 14 and was the comment out of date?

Reply:
sort of, the comment is accurate because this config should reflect prod. This PR should have been merged the same week Wei updated the node version in the `Dockerfile` (for some reason, that's how we do it... although it'd be slightly better to update all node version references at once)